peertalk/PTUSBHub.m

```diff
@@ -211,7 +211,7 @@ - (void)listenOnQueue:(dispatch_queue_t)queue onStart:(void(^)(NSError*))onStart
 }
 
 
-- (void)connectToDevice:(NSNumber*)deviceID port:(int)port onStart:(void(^)(NSError*, dispatch_io_t))onStart onEnd:(void(^)(NSError*))onEnd {
+- (void)connectToDevice:(NSNumber*)deviceID port:(uint32_t)port onStart:(void(^)(NSError*, dispatch_io_t))onStart onEnd:(void(^)(NSError*))onEnd {
```

Comment:
My issue with `int` here is that there isn't a reason to ever pass a negative value as the port, therefore the user facing interface shouldn't require the implementation to handle a potential input that is unnecessary. As far as I can tell passing a negative value as the port has undefined behavior.

I have no problem with using `unsigned int` here. The reasoning behind using `uint32_t` (which looking back should be `uint16_t`) was because that appears to be the standard way throughout the rest of the source. I value consistency more highly then correctness.